api/internal/localizer/locloader_test.go

```diff
@@ -37,6 +38,9 @@ func TestLocalLoadNewAndCleanup(t *testing.T) {
 
 	var buf bytes.Buffer
 	log.SetOutput(&buf)
+	defer func() {
```

Comment:
Can we do this for the other occurrence of `log.SetOutput` in this file: https://github.com/kubernetes-sigs/kustomize/pull/5048/files#diff-d9602d072929ae47846f3e890b067a08f8306f2135fcb5fd2b1697c80a0cbb9cR229

Thank you for taking care of this!

Reply:
Done!